tests/scripts/all.sh

```diff
@@ -34,13 +34,26 @@ MEMORY=0
 SHORT=0
 FORCE=0
 
+OPENSSL="openssl"
```

Comment:
Also, by adding environment variables for OpenSSL and GNUTLS, please also add outputting the version of those legacy versions to the `output_env.sh` script, if they are defined.